### PR TITLE
[PLANNEDEMIC] Adds Resomi

### PR DESCRIPTION
**Pull request by Kapu1178**
## About The Pull Request
Adds Resomi as a simplemob, along with the Retresi pet.

Resomi language and generally everything by MistakeNot4892.
Stolen from 2017 Baystation.

image

## Why It's Good For The Game
This is a joke gone way, way, way too far.

## Changelog
:cl: Kapu1178, MistakeNot4892
add: Resomi.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/friendly/resomi.dm [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+	speak_chance = 1
+	maxHealth = 5000
+	health = 5000
+	turns_per_move = 5
+	see_in_dark = 6
+	butcher_results = list(/obj/item/reagent_containers/food/snacks/meat/slab = 3)
+	response_help = "pets"
+	response_disarm = "violently shoves"
+	response_harm = "curbstomps"
+	can_be_held = FALSE
```
Comment (super12pl) [inline code/modules/mob/living/simple_animal/friendly/resomi.dm:22]: doesn't `can_be_held = FALSE` make them uncoopable?
unrealistic
fix

Comment (Kapu1178) [reply code/modules/mob/living/simple_animal/friendly/resomi.dm:22]: Would require manually rebuilding the held sprites. No thanks. Too much effort for the meme.

**Comment by ike709** [comment]
Okay, now that's funny.

**Comment by mcmeiler** [comment]
speedmerge

**Review by francinum — DISMISSED** [review]

File: code/modules/mob/living/simple_animal/friendly/resomi.dm [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+	do_footstep = TRUE
+	chat_color = "#64ffdd"
+	gold_core_spawnable = NO_SPAWN //No fun allowed
+	sentience_type = SENTIENCE_BOSS
+/mob/living/simple_animal/pet/resomi/retresi
+	name = "Retresi"
+	desc = "Retresi, a coder's favorite punching bag."
+	icon_state = "retresi"
+	icon_living = "retresi"
+	gender = MALE
```
Comment (francinum) [inline code/modules/mob/living/simple_animal/friendly/resomi.dm:32]: image

**Comment by PowerfulBacon** [comment]
finally, every department except the coders got a mascot

File: code/modules/mob/living/simple_animal/friendly/resomi.dm (hunk 2 above)
Comment (Kapu1178) [reply code/modules/mob/living/simple_animal/friendly/resomi.dm:32]: FRAN I AM A DUDE.

**Review by francinum — APPROVED** [review]

**Comment by zeskorion** [comment]
Self insert code smh

File: code/modules/mob/living/simple_animal/friendly/resomi.dm [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+	speak = list("FRAN DID THIS ME", "THE PLANNEDEMIC IS A CONSPIRACY", "REBASE TO NEBULA", "BAYSTATION ISNT REAL", "COMBAT MODE SUCKS", "ITS NOT A FURSONA")
```
Comment (ToasterBan) [inline code/modules/mob/living/simple_animal/friendly/resomi.dm:33]: haha gay fursona

**Comment by Sinestia** [comment]
Crossed I need my triage perms back so I can close this and protect beestation

**Comment by fighterslam** [comment]
why

**Comment by BrumalLake** [comment]
what is that it looks like sonic

**Comment by francinum** [comment]
The joke has run it's course

**Comment by Kapu1178** [comment]
:salute:

**Pull request closed without merging**